packages/api-server/src/methods/error.ts

```diff
@@ -32,6 +32,19 @@ export class InvalidParamsError extends RpcError {
   constructor(message: string) {
     super(INVALID_PARAMS, message);
   }
+
+  padErrorContext(context: string): InvalidParamsError {
```

Comment:
```suggestion
  padContext(context: string): InvalidParamsError {
```
It's already an error object